tests/out/spv/boids.spvasm

```diff
@@ -9,20 +9,19 @@ OpMemoryModel Logical GLSL450
 OpEntryPoint GLCompute %46 "main" %43
 OpExecutionMode %46 LocalSize 64 1 1
 OpSource GLSL 450
-OpName %3 "NUM_PARTICLES"
```

Comment:
This name for the constant goes away with this PR, but for some other languages (GLSL, for example), the `NUM_PARTICLES` constant gets added to the output. What's going on here?

Reply:
We are now iterating over `const_expressions` instead of `constants` which means the name is not being written anymore...

I'll come up with a better solution.